packages/docusaurus-plugin-content-docs/src/types.ts

```diff
@@ -69,6 +71,8 @@ export type VersionOptions = {
   path?: string;
   label?: string;
   banner?: VersionBanner;
+  badge?: boolean;
```

Comment:
```suggestion
  showBadge?: boolean;
```

`badge` is a noun but `showBadge` sounds more boolean

Reply:
agree, but I plan later to make it a `string | boolean` so for better extensibility without breaking change I prefer to keep `badge` here